### PR TITLE
infra: switch travis jobs to github workflow

### PR DESCRIPTION
**Pull request by nmancus1**
We need to temporarily move these CI jobs to a github workflow until we restore travis.

### CODE REVIEW

File: .ci/validation.sh [hunk 1]
```diff
@@ -201,7 +201,7 @@ no-error-xwiki)
   cd ..
   checkout_from https://github.com/xwiki/xwiki-platform.git
   cd .ci-temp/xwiki-platform
-  git checkout "1d243e9a31589c7a2b2b730bd5e""ad67cb989ca8b"
+#  git checkout "1d243e9a31589c7a2b2b730bd5e""ad67cb989ca8b"
```
Comment (nmancus1) [inline .ci/validation.sh:204]: Do we still need to use this specific commit?

Comment (rnveach) [reply .ci/validation.sh:204]: If the external repo doesn't not enforce a stable build, we use specific commits that are stable. I thought xwiki was one of those.

File: .github/workflows/travis-workflow.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+name: "no-exception testing"
```
Comment (strkkk) [inline .github/workflows/travis-workflow.yml:1]: name of the file is a bit confusing

Comment (nmancus1) [reply .github/workflows/travis-workflow.yml]: Sure is. Fixed.

File: .ci/validation.sh (hunk 1 above)
Comment (nmancus1) [reply .ci/validation.sh]: Updated.

**Review by strkkk — APPROVED** [review]

**Comment by strkkk** [comment]
@nmancus1 there is a failure in xwiki, is it expected?

**Comment by nmancus1** [comment]
> there is a failure in xwiki, is it expected?

No, I used what I thought was the last commit that passed for me, but I was wrong. Updated with the correct commit.

**Review by romani — APPROVED** [review]
@nmancus1 ,
it would be better to split such big job (21 min) to multiple each execution separately, it will make it more quick by parallelism and easy to restart only single execution.

**Comment by romani** [comment]
I still have discussion with Travis support, there are some progress, Travis might return soon, but we will find for it new tasks.

**Pull request merged**